pkg/local_workflows/output_workflow_test.go

```diff
@@ -338,14 +338,7 @@ func getSarifInput() sarif.SarifDocument {
 									Level: "error",
 								},
 								Properties: sarif.RuleProperties{
-									Tags: []string{"tag1", "tag2"},
-									ShortDescription: sarif.ShortDescription{
-										Text: "Rule 1 description",
-									},
-									Help: struct {
-										Markdown string `json:"markdown"`
-										Text     string `json:"text"`
-									}{Markdown: "", Text: ""},
+									Tags:               []string{"tag1", "tag2"},
```

Comment:
These fields are already on the rule itself, so I don't think they're required in the property bag

Reply:
Question: did this break something?

Reply:
Do you mean by manually testing GAF? Here are screenshots.

- Without FF
![Screenshot 2024-04-25 at 17 05 53](https://github.com/snyk/go-application-framework/assets/81559517/386a6e57-0cdc-42be-82f8-7d187defd0df)

- With FF
![Screenshot 2024-04-25 at 17 06 12](https://github.com/snyk/go-application-framework/assets/81559517/f39d28ab-724f-4756-acc7-dc29b6472f6b)

Reply:
I mean it is just some test data for a unit test, so I was wondering if it hurts that it is there.

Reply:
I just think that the test case has nothing to do with your changes, so maybe we don't need to remove the data, but actually if the tests still pass it is probably obsolete.

Reply:
Ah I get you now. I changed the test case because the fields have been removed from the type. So the tests were failing as a result

Reply:
Here are the changes in `code-client-go`, for reference: https://github.com/snyk/code-client-go/pull/37